neqo-bin/src/client/mod.rs

```diff
@@ -245,13 +251,13 @@ impl Args {
                     self.method = String::from("POST");
                 }
             }
-            "handshake"
-            | "transfer"
-            | "retry"
-            | "ecn"
-            | "rebind-port"
-            | "rebind-addr"
-            | "connectionmigration" => {}
+            "handshake" | "transfer" | "retry" | "ecn" => {}
+            "rebind-port" | "rebind-addr" | "connectionmigration" => {
+                if self.cid_len == 0 {
+                    qinfo!("Rebind/migration test won't work with len-0 CID; overwriting to 8");
+                    self.cid_len = 8;
+                }
```

Comment:
We should be able to do client-side address rebinding if the client uses a zero-length CID.  Which side is doing the rebinding again?

Reply:
I need to check tomorrow, but I think the issue is that we don't issue a new connection ID on the client when the server retires a path and the connection ID used for it.

Reply:
Yes, what's happening is that with a client using a len-0 CID, when the server has validated the new path after a rebinding event, it will retire the old path and hence send a `RETIRE_CONNECTION_ID` for CID number 0. After the client receives and processes that frame, it will no longer accept any packets from the server (`Dropped received packet: Invalid DCID CID [0]: ;`).

Reply:
See #2182.